avalon/nuke/pipeline.py

```diff
@@ -279,18 +266,9 @@ def _install_menu():
         publish,
         workfiles,
         loader,
-        sceneinventory,
-        libraryloader
```

Comment:
Maybe we should keep `libraryloader` here (if is not overriden with ours menu in pype).

Reply:
we override menu completery